azure-pipelines.yml

```diff
@@ -49,6 +49,19 @@ stages:
       archive_pytests: true
       archive_gcov: true
 
+- stage: BuildAsan
+
```

Comment:
Does the BuildAsan stage have dependency on Build stage, if not, you can add the dependsOn:[] to skip it, it will run in parallel with Build stage.
```
- stage: BuildAsan
  dependsOn: []
```

Reply:
Good idea, thanks. I've updated the PR.